src/EventStore.Core/ClusterVNode.cs

```diff
@@ -1816,10 +1816,14 @@ public class ClusterVNode<TStreamId> :
 				Log.Information("Skipping reload of certificates since TLS is disabled.");
 				return;
 			}
-
-			if (_certificateProvider?.LoadCertificates() == LoadCertificateResult.VerificationFailed){
+			
+			var prevthumbprint = _certificateProvider.Certificate?.Thumbprint;
+			var temp = new OptionsCertificateProvider(options);
+			if (temp?.LoadCertificates(prevthumbprint) == LoadCertificateResult.VerificationFailed){
```

Comment:
Have you considered passing the options to the `LoadCertificates` call?

Reply:
I think passing options to LoadCertificates() call would not help. The only reason to come up with a new implementation of the LoadCertificates() method was because the previous thumbprint value was lost on loading the new certs, so came up with this new implementation. Before 22.10 and before DevCertificates, LoadCertificates() was not implemented in a separate class, so there was no need of tracking the old certificate thumbprint through a parameter.

Reply:
Implemented the LoadCertificates() method by passing options as parameter along with the previous thumbprint.